### PR TITLE
tests/bb: update configMinVersion to reflect lowest compatible config version

### PR DESCRIPTION
**Pull request by Rahuls0720**
Updates the configMinVersion to reflect the lowest compatible config version.

** Opened this PR so I can run the BB test on jenkins since my system has mounting/unmounting issues

### CODE REVIEW

File: tests/positive/files/file.go [hunk 1]
```diff
@@ -129,7 +129,7 @@ func UserGroupByName() types.Test {
 			Contents: "example file\n",
 		},
 	})
-	configMinVersion := "2.1.0"
+	configMinVersion := "2.0.0"
```
Comment (ajeddeloh) [inline tests/positive/files/file.go]: 2.0.0 does not have the name key for users/groups

File: tests/positive/files/file.go [hunk 2]
```diff
@@ -220,7 +220,7 @@ func ForceFileCreationNoOverwrite() types.Test {
 			Contents: "asdf\nfdsa",
 		},
 	})
-	configMinVersion := "2.2.0"
+	configMinVersion := "2.1.0"
```
Comment (ajeddeloh) [inline tests/positive/files/file.go]: why not 2.0.0

File: tests/positive/partitions/verification.go [hunk 3]
```diff
@@ -92,7 +92,7 @@ func VerifyBaseDisk() types.Test {
 			}]
 		}
 	}`
-	configMinVersion := "2.1.0"
+	configMinVersion := "2.0.0"
```
Comment (ajeddeloh) [inline tests/positive/partitions/verification.go]: 2.0.0 configs don't have `guid`

File: tests/positive/files/file.go (hunk 1 above)
Comment (Rahuls0720) [reply tests/positive/files/file.go]: Fixed

File: tests/positive/files/file.go (hunk 2 above)
Comment (Rahuls0720) [reply tests/positive/files/file.go]: Fixed

File: tests/positive/partitions/verification.go (hunk 3 above)
Comment (Rahuls0720) [reply tests/positive/partitions/verification.go]: Fixed

**Pull request merged**